Dockerfile.apache

```diff
@@ -8,12 +8,20 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-FROM docker.io/httpd:2.4.43-alpine
+FROM node:8.16.2 as builder
```

Comment:
where is build that image from CI ?

Reply:
I do not understand your question. Can you please rephrase?